src/main/java/com/getbase/jenkins/plugins/metrics/history/influxdb/generators/JenkinsBasePointGenerator.java

```diff
@@ -61,9 +89,10 @@ public JenkinsBasePointGenerator(Run<?, ?> build) {
                 .addField(BUILD_DURATION, duration)
                 .addField(QUEUING_DURATION, action.getQueuingDurationMillis())
                 .addField(TOTAL_DURATION, duration + action.getQueuingDurationMillis())
-                .addField(BUILD_STATUS_MESSAGE, build.getBuildStatusSummary().message);
+                .addField(BUILD_STATUS_MESSAGE, build.getBuildStatusSummary().message)
```

Comment:
AFAIK `getBuildStatusSummary()` may return null, which will cause this line to blow up. We should have a check for that somewhere to avoid it.

Reply:
@karolgil fixed